render/wgpu/src/context3d/mod.rs

```diff
@@ -286,6 +290,14 @@ impl WgpuContext3D {
                 .as_ref()
                 .expect("Missing compiled pipeline"),
         );
+        if let Some(rect) = &self.scissor_rectangle {
+            pass.set_scissor_rect(
```

Comment:
wgpu will panic if these bounds exceed the render attachment. I don't think we're checking for this, so what does Flash do?

Reply:
Flash throws an error during `drawTriangles`, but only if `Context3D.enableErrorChecking` is enabled. For now, I think we could just skip setting the scissor rect on the wgpu side, and log a warning.

Reply:
I've added in an error log when the rectangle is too large.